advising/urls.py

```diff
@@ -36,10 +36,10 @@
     # Get the courses for the requesting user
     url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/$',
         advising.views.RegisteredCoursesView.as_view()),
-    # Get the courses for the student with this JHED
-    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<jhed>.+)/$',
+    # Get the courses for the student with this JHED (Empty value for tt_name must be passed)
+    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})//(?P<jhed>.+)/$',
         advising.views.RegisteredCoursesView.as_view()),
     # Get the courses for the student with this JHED and verify it against this student's timetable
-    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<jhed>.+)/(?P<tt_name>.+)/$',
+    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<tt_name>.+)/(?P<jhed>.+)/$',
```

Comment:
Did this actually work? 
I was thinking to swap the urls like this:
```
    # Get the courses for the student with this JHED and verify it against this student's timetable
    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<jhed>.+)/(?P<tt_name>.+)/$',
        advising.views.RegisteredCoursesView.as_view()),
    # Get the courses for the student with this JHED
    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/(?P<jhed>.+)/$',
        advising.views.RegisteredCoursesView.as_view()),
    # Get the courses for the requesting user
    url(r'^advising/sis_courses/(?P<sem_name>.+)/(?P<year>[0-9]{4})/$',
        advising.views.RegisteredCoursesView.as_view())
```
So that the url with 4 parameters is used first.

Reply:
hmm in not sure if it mtaches in order like that but i can try

Reply:
what is the use case for the last URL? Why don't we just use the first two?

Reply:
That last URL was left in for backwards compatibility in case anyone was still using the endpoint without specifying the JHED. I can remove it (and the corresponding unit test) if it's no longer used.